core/server/master/src/test/java/alluxio/master/metastore/InodeStoreTest.java

```diff
@@ -38,52 +40,106 @@
+    File confFile = new File(sDir + CONF_NAME);
+    writeStringToFile(confFile, ROCKS_CONFIG, (Charset) null);
+
     return Arrays.asList(
         lockManager -> new HeapInodeStore(),
-        lockManager -> new RocksInodeStore(dir),
-        lockManager -> new CachingInodeStore(new RocksInodeStore(dir), lockManager));
+        lockManager -> new RocksInodeStore(sDir),
+        lockManager -> new CachingInodeStore(new RocksInodeStore(sDir), lockManager));
   }
 
   @Rule
-  public ConfigurationRule mConf =
-      new ConfigurationRule(PropertyKey.MASTER_METASTORE_INODE_CACHE_MAX_SIZE,
-          CACHE_SIZE, ServerConfiguration.global());
+  public ConfigurationRule mConf = new ConfigurationRule(new HashMap<PropertyKey, Object>() {
+    {
+      put(PropertyKey.MASTER_METASTORE_INODE_CACHE_MAX_SIZE, CACHE_SIZE);
+    }
+  }, ServerConfiguration.global());
 
   private final MutableInodeDirectory mRoot = inodeDir(0, -1, "");
 
-  private final InodeStore mStore;
-  private final InodeLockManager mLockManager;
+  private final Function<InodeLockManager, InodeStore> mStoreBuilder;
+  private InodeStore mStore;
+  private InodeLockManager mLockManager;
 
   public InodeStoreTest(Function<InodeLockManager, InodeStore> store) {
+    mStoreBuilder = store;
+  }
+
+  @Before
+  public void before() {
     mLockManager = new InodeLockManager();
-    mStore = store.apply(mLockManager);
+    mStore = mStoreBuilder.apply(mLockManager);
   }
 
   @After
   public void after() {
     mStore.close();
   }
 
+  @Test
+  public void rocksConfigFile() throws Exception {
+    assumeTrue(mStore instanceof RocksInodeStore || mStore instanceof CachingInodeStore);
+    // close the store first because we want to reopen it with the new config
+    mStore.close();
+    try (AutoCloseable ignored = new ConfigurationRule(new HashMap<PropertyKey, Object>() {
+      {
+        put(PropertyKey.ROCKS_INODE_CONF_FILE, sDir + CONF_NAME);
+      }
+    }, ServerConfiguration.global()).toResource()) {
+      before();
+      writeInode(mRoot);
+      assertEquals(Inode.wrap(mRoot), mStore.get(0).get());
+    }
+  }
+
+  @Test
+  public void rocksInvalidConfigFile() throws Exception {
+    assumeTrue(mStore instanceof RocksInodeStore || mStore instanceof CachingInodeStore);
+    // close the store first because we want to reopen it with the new config
+    mStore.close();
+    // write an invalid config
+    String path = sDir + CONF_NAME + "invalid";
+    File confFile = new File(path);
+    writeStringToFile(confFile, "Invalid config", (Charset) null);
+
+    try (AutoCloseable ignored = new ConfigurationRule(new HashMap<PropertyKey, Object>() {
+      {
+        put(PropertyKey.ROCKS_INODE_CONF_FILE, path);
+      }
+    }, ServerConfiguration.global()).toResource()) {
+      assertEquals(RocksDBException.class,
+          assertThrows(RuntimeException.class, this::before).getCause().getClass());
```

Comment:
```suggestion
      RuntimeException exception = assertThrows(RuntimeException.class, this::before);
      assertEquals(RocksDBException.class, exception.getCause().getClass());
```
Same comment as in `BlockStoreTest`.